src/DoctrineMongoODMModule/Module.php

```diff
@@ -143,6 +143,7 @@ public function getServiceConfig()
             ),
             'aliases' => array(
                 'Doctrine\ODM\Mongo\DocumentManager' => 'doctrine.documentmanager.odm_default',
+                'Doctrine\ODM\MongoDB\DocumentManager' => 'doctrine.documentmanager.odm_default',
```

Comment:
Y not? I use `$serviceLocator->get(EntityManager::class);` everywhere when using ORM